tests/unit/preprocessor/RandomFourierGaussPreproc_unittest.cc

```diff
@@ -0,0 +1,66 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Author: Nanubala Gnana Sai
+ */
+
+#include <gtest/gtest.h>
+#include <shogun/features/DenseFeatures.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <shogun/mathematics/NormalDistribution.h>
+#include <shogun/mathematics/linalg/LinalgNamespace.h>
+#include <shogun/preprocessor/RandomFourierGaussPreproc.h>
+#include <math.h>
+using namespace shogun;
+
+class RFGPTest : public ::testing::Test
+{
+public:
+	virtual void SetUp()
+	{
+		SGMatrix<float64_t> mat(num_features, num_vectors);
+		linalg::range_fill(mat, 1.0);
+		auto features = std::make_shared<DenseFeatures<float64_t>>(mat);
+
+		preproc = std::make_shared<RandomFourierGaussPreproc>();
+		preproc->put(param0, seed);
+		preproc->put(param1, target_dim);
+		preproc->put(param2, log_width);
+		preproc->fit(features);
+	}
+	virtual void TearDown()
+	{
+	}
+
+protected:
+	const int32_t seed = 100;
+	const index_t num_vectors = 5;
+	const index_t num_features = 3;
+	const index_t target_dim = 300;
+	const float64_t log_width = log(100);
```

Comment:
way too large

Reply:
also please `std::log`

Reply:
@gf712 we see, to have a `using namespace std` in some header file. You have an idea where that happened?